cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/MultivariateAnomalyDetection.scala

```diff
@@ -371,15 +378,17 @@ trait MADBase extends HasOutputCol
 
 }
 
-object FitMultivariateAnomaly extends ComplexParamsReadable[FitMultivariateAnomaly] with Serializable
+object SimpleFitMultivariateAnomaly extends ComplexParamsReadable[SimpleFitMultivariateAnomaly] with Serializable
 
-class FitMultivariateAnomaly(override val uid: String) extends Estimator[DetectMultivariateAnomaly]
+class SimpleFitMultivariateAnomaly(override val uid: String) extends Estimator[SimpleDetectMultivariateAnomaly]
   with MADBase {
   logClass()
 
-  def this() = this(Identifiable.randomUID("FitMultivariateAnomaly"))
+  def this() = this(Identifiable.randomUID("SimpleFitMultivariateAnomaly"))
+
+  def urlPath: String = "anomalydetector/v1.1/multivariate/models"
 
-  def urlPath: String = "anomalydetector/v1.1-preview/multivariate/models"
+  val dataSchema = "OneTable"
```

Comment:
Sorry but What does this mean lol?

Reply:
v1.1 adds new schema, and they support either inputting `OneTable` or `MultiTable`. OneTable is just one csv file containing both timestamp column and feature columns, and MultiTable is the zip file what we did before.